src/styles/card.css

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  padding: 0 16px;
```

Comment:
You can set up paddings for the whole card, why 0?

Reply:
Why don't you fix @ivan-kadykalo comments?